mock-core-configs/etc/mock/templates/alma-8.tpl

```diff
@@ -0,0 +1,119 @@
+config_opts['chroot_setup_cmd'] = 'install tar gcc-c++ redhat-rpm-config redhat-release which xz sed make bzip2 gzip gcc coreutils unzip shadow-utils diffutils cpio bash gawk rpm-build info patch util-linux findutils grep'
+config_opts['dist'] = 'el8'  # only useful for --resultdir variable subst
+config_opts['releasever'] = '8'
+config_opts['package_manager'] = 'dnf'
+config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
+config_opts['bootstrap_image'] = 'alma:8'
+
+
+config_opts['dnf.conf'] = """
+[main]
+keepcache=1
+debuglevel=2
+reposdir=/dev/null
+logfile=/var/log/yum.log
+retries=20
+obsoletes=1
+gpgcheck=0
+assumeyes=1
+syslog_ident=mock
+syslog_device=
+metadata_expire=0
+mdpolicy=group:primary
+best=1
+protected_packages=
+module_platform_id=platform:el8
+user_agent={{ user_agent }}
+
+# almalinux.repo
+#
+# The mirror system uses the connecting IP address of the client and the
+# update status of each mirror to pick mirrors that are updated to and
+# geographically close to the client.  You should use this for AlmaLinux updates
+# unless you are manually picking other mirrors.
+#
+[baseos]
+name=AlmaLinux 8.3 - BaseOS
+baseurl=https://repo.almalinux.org/almalinux/8/BaseOS/$basearch/os/
+enabled=1
+gpgcheck=0
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux
+
+[appstream]
+name=AlmaLinux 8.3 - AppStream
+baseurl=https://repo.almalinux.org/almalinux/8/AppStream/$basearch/os/
+enabled=1
+gpgcheck=0
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux
+
+[powertools]
+name=AlmaLinux 8.3 - PowerTools
+baseurl=https://repo.almalinux.org/almalinux/8/PowerTools/$basearch/os/
+enabled=1
+gpgcheck=0
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux
+
+[extras]
+name=AlmaLinux 8.3 - Extras
+baseurl=https://repo.almalinux.org/almalinux/8/extras/$basearch/os/
+enabled=1
+gpgcheck=0
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux
```

Comment:
Several issues here:

* This references AlmaLinux 8.3 but doesn't set a fixed path, should it be "AlmaLinux 8"?
* GPG check is turned off
* Path to the GPG key file is invalid, the key needs to be added to [xsuchy/distribution-gpg-keys](https://github.com/xsuchy/distribution-gpg-keys) like all the others have